internal/conf/conf.go

```diff
@@ -94,3 +107,35 @@ func initEtcdConfig(conf Etcd) {
 		Prefix:   prefix,
 	}
 }
+
+func initLogConfig(conf Log) {
+	level := conf.Level
+	if level == "" {
+		level = "warn"
```

Comment:
It would be better to let the default log level be equal to the log level in the default log level file. Otherwise, people will be confused.